seed/static/seed/js/controllers/data_quality_admin_controller.js

```diff
@@ -352,6 +373,10 @@ angular.module('BE.seed.controller.data_quality_admin', [])
           show_configuration_errors(misconfigured_rules);
         }
 
+        // Find duplicate rules and trigger warnings
+        $scope.is_duplicate = ($scope.duplicate_rule_keys.length > 1);
+        if ($scope.is_duplicate) return Notification.error({message: "Duplicate rules detected.", delay: 10000});
+
```

Comment:
I think you want to move the notification logic into the show configuration errors